test/functional/device/calendar-e2e-specs.js

```diff
@@ -1,40 +1,43 @@
 import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
-import { TESTAPP_CAPS } from '../desired';
+import { TESTAPP_CAPS, amendCapabilities } from '../desired';
 import { initSession, deleteSession, MOCHA_TIMEOUT } from '../helpers/session';
 
 
 chai.should();
 chai.use(chaiAsPromised);
 
 if (!process.env.REAL_DEVICE && !process.env.CLOUD) {
-  describe('XCUITestDriver - calendar', function () {
+  describe('XCUITestDriver - calendar', function() {
     this.timeout(MOCHA_TIMEOUT);
 
     let driver, caps;
 
-    beforeEach(function () {
+    beforeEach(function() {
       caps = Object.assign({}, TESTAPP_CAPS);
```

Comment:
not sure if Object.assign is needed there. TESTAPP_CAPS are anyway immutable

Reply:
fixed https://github.com/appium/appium-xcuitest-driver/pull/1461/commits/869d928778337112826a0bb39d8dd6b408c9f72e